### PR TITLE
[PDR-956] Update researcher generator to support pre-DSV2 records.

### PR DESCRIPTION
**Pull request by robabram**
## Description of changes/additions
Update Workbench Researcher PDR generator to support pre-DSV2 survey records.

## Tests
- [x] unit tests : Added new unittest.

### CODE REVIEW

File: rdr_service/resource/generators/workbench.py [hunk 1]
```diff
@@ -210,6 +210,15 @@ def make_resource(self, src_pk_id, backup=False):
         :param backup: if True, get from backup database instead of Primary.
         :return: resource object
         """
+
+        def v_to_e(val, enum_=WorkbenchResearcherYesNoPreferNot):
```
Comment (dtharpeuno) [inline rdr_service/resource/generators/workbench.py]: Can we rename this to encompass the action, maybe `value_to_enum`?

Comment (robabram) [reply rdr_service/resource/generators/workbench.py]: Done

File: tests/resource_tests/generator_tests/test_workbench.py [hunk 2]
```diff
@@ -82,3 +189,15 @@ def test_workbench_researcher_generator(self):
...
         self.assertEqual(data['degrees'], [{'degree': 'PHD', 'degree_id': 1}, {'degree': 'MBA', 'degree_id': 8}])
+
+    def test_workbench_researcher_dsv2_generator(self):
+        self.session.add(self.researcher2)
+        self.session.add(self.researcher3)
+        self.session.commit()
+
+        gen = WBResearcherGenerator()
+        res = gen.make_resource(2)
+        self.assertIsNotNone(res)
```
Comment (pegbertsch) [inline tests/resource_tests/generator_tests/test_workbench.py:200]: Besides `assertIsNotNone(res)`, are there any other finer-grained assertions to validate on the PDR generator record?

**Review by pegbertsch — APPROVED** [review]
Approving based on functional changes looking ok.  

Optionally/minor, wondering if the unit test case was meant just to expose unexpected exceptions thrown by the generator, given there isn't any validation on the data returned from the `make_resource` call except that it exists.

File: tests/resource_tests/generator_tests/test_workbench.py (hunk 2 above)
Comment (robabram) [reply tests/resource_tests/generator_tests/test_workbench.py:200]: The test was just to make sure that both a pre-DSV2 and post-DSV2 record won't throw an exception in the generator.

**Review by dtharpeuno — APPROVED** [review]

**Pull request merged**